tests/kernel/fatal/message_capture/README

```diff
@@ -0,0 +1,5 @@
+This test case verifies that kernel fatal error logging message can be captured.
+
+When the fatal error is triggered, if the debugging message function is turned on,
+the system can capture the print information.
+
```

Comment:
Please put the comments in doxgen tag.

Reply:
I just copy the file “kernel/fatal/README” and changed some words here. The original file did not see any use of doxgen tag. And I find the README file in zephyrproject, other README file is also like this.